scripts/flatten.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+outdir="build/flattened"
+
+function flatten() {
+  ./node_modules/.bin/truffle-flattener "$1" --output "$2"
+}
+
+for inpath in contracts/*.sol; do
+  filename=$(basename "$inpath" .sol)
+
+  mkdir -p "$outdir"
```

Comment:
Move this before the loop?
I mean, it will exit with no error if the dir already exists, but I think cleaner to put it as an initialization step

Reply:
oh yeah, good point. That was a mistake